collections/_article/dev-snapshot-godot-4-2-dev-3.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 3"
+excerpt: "One month into the release cycle for Godot 4.2, things are well under way. This third dev snapshot features a lot of improvements to C# and rendering."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-3.webp
+image_caption_title: "Easy Charts"
+image_caption_description: "A plotting addon by fenix-hub"
+date: 2023-08-11 14:00:00
+---
+
+Development is well under way for our next milestone, Godot 4.2. After our [dev 2 snapshot](/article/dev-snapshot-godot-4-2-dev-2/), we kept merging a number of new features and important bug fixes, so we're ready for a third dev snapshot to put those under broader user testing. It's been around a month since we started this release cycle, with roughly two months to go for 4.2 beta 1, and a fourth month to reach stable in early November 2023.
+
+I'd like to outline the amazing work done by the [Production team](https://godotengine.org/teams/#production), and especially by Yuri Sizov, who took care of PR reviews, merges, and the whole release process while I was on holiday in the second half of July. Godot 4.1.1, the first two dev snapshots for 4.2, and 4.0.4, were all fully handled by Yuri, while I used to be the sole responsible for making and publishing releases. We're now fully sharing the role of release manager, and that should enable us to keep this sustained pace with a high quality standard.
```

Comment:
"I used to be the sole responsible for" would be better as e.g. "I used to have sole responsibility for" or "I used to be solely responsible for".

Reply:
"solely responsible" means that it would be my only responsibility, no? What I mean is that I was the only one having that responsibility (but I had and still have dozen other responsibilities in this project).

Reply:
Hmm, it maybe could be confusing. I'm not sure of a better choice of words at the moment. But "the sole responsible" is not right - unless you meant "the soul responsible"!